windows.sh

```diff
@@ -33,6 +33,7 @@ fi
 if [ ! -f /usr/bin/ansible ]; then
   echo "Installing pip..."
   sudo apt-get -y install python-pip
+  sudo apt-get -y update
```

Comment:
Seems like it should be before. Are you testing from scratch to test? Anyway, let's put it before 👍 and we're good